### PR TITLE
add Headline, Menuicon, Cocktailpic, Cocktailname, Dummy Seach

### PR DESCRIPTION
**Pull request by AndreHaupt85**

### CODE REVIEW

**Review by K-Juno — APPROVED** [review]
Great job! Happy Coding!

File: pages/index.js [hunk 1]
```diff
@@ -1,7 +1,93 @@
+import Image from "next/image";
+import styled from "styled-components";
+import icon from "../src/cocktail_icon.svg"
+import cocktail from "../src/old_fashioned.jpeg"
+
+
+
+console.log(icon);
```
Comment (MerleMak) [inline pages/index.js:8]: Please delete any console.log's prior to commiting :)

File: pages/index.js [hunk 2]
```diff
@@ -1,7 +1,93 @@
...
+          alt="menu icon"
+        />
+        <Cocktailname>
+          Old Fashioned
+        </Cocktailname>
+      </Cocktail>
+      <Search>
+        <form>
+          <label for="quicksearch" ></label>
+          <input type="text" id="quicksearch" />
```
Comment (MerleMak) [inline pages/index.js:36]: So far you didn't add any functionality to your search-field. We want only commit features that have a functionality, so you could still leave out this input field until you also want to add functionality :)

File: pages/index.js [hunk 3]
```diff
@@ -1,7 +1,93 @@
...
+
+const Headline = styled.h1`
+color: #FFBA08;
+background: #7D451B;
+padding: 0;
+margin: 0;
+font-size: 40px;
+`
+
+// Random Cocktail
```
Comment (MerleMak) [inline pages/index.js:65]: Please delete any comments before commiting the code :)

File: pages/index.js [hunk 4]
```diff
@@ -1,7 +1,93 @@
...
+height: 80vh;
+padding-top: 20px;
+padding-left: 20px;
+padding-right: 20px;
+`
+
+const Cocktailname = styled.p`
+font-size: 45px;
+`
+// Quicksearch
```
Comment (MerleMak) [inline pages/index.js:83]: see above :)

**Review by MerleMak — COMMENTED** [review]
i made some small remarks, good job! :)

**Pull request merged**